docs/core/tools/sdk-errors/netsdk1022.md

```diff
@@ -11,7 +11,7 @@ f1_keywords:
 
 **This article applies to:** ✔️ .NET 2.1.100 SDK and later versions
 
-Starting in Visual Studio 15.3, the .NET SDK automatically includes items from the project directory by default.  This includes `Compile` and `Content` targets.  This should greatly clean up your project file and reduce the complexity you have there.
+Starting in Visual Studio / MSBuild 15.3, the .NET SDK automatically includes items from the project directory by default.  This includes `Compile` and `Content` targets.  This should greatly clean up your project file and reduce the complexity you have there.
```

Comment:
Should we use "version" here like we normally do for VS? Also, 15.3 is 2017.
```suggestion
Starting in Visual Studio 2017 / MSBuild version 15.3, the .NET SDK automatically includes items from the project directory by default.  This includes `Compile` and `Content` targets.  This should greatly clean up your project file and reduce the complexity you have there.
```

Reply:
Yes, thanks!